src/components/core/GalleryLink/GalleryLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode } from 'react';
+import { Link } from '@reach/router';
+import styled from 'styled-components';
+
+type Props = {
+  to?: string;
+  href?: string;
+  children: ReactNode;
+  underlined?: boolean;
+  underlineOnHover?: boolean;
+};
+
+export default function GalleryLink({ to, href, children, underlined = true, underlineOnHover = false }: Props) {
+  if (!to && !href) {
+    throw new Error('no link provided for GalleryLink');
+  }
+
+  if (to) {
+    return (
+      <StyledLink to={to} underlined={underlined} underlineOnHover={underlineOnHover}>
+        {children}
+      </StyledLink>
+    );
+  }
+
+  if (href) {
+    return (
+      <StyledAnchor href={href} target="_blank" rel="noopener noreferrer" underlined={underlined} underlineOnHover={underlineOnHover}>{children}</StyledAnchor>
```

Comment:
side note: noticed our prettier max line length went away. we should bring that back